### PR TITLE
Portfolio project popup

### PR DESCRIPTION
**Pull request by yodit93**
### In this project:

- [ ] I have added all the project information inside an object.
- [ ] I have kept all project objects inside an array.
- [ ] I recreated the project section of my portfolio using java-script.
- [ ] I have added a popup window  for the projects in response to the click event on mobile and desktop version of the portfolio.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    name: ESLint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup ESLint
+        run: |
+          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
```
Comment (Olamarx) [inline .github/workflows/linters.yml:59]: - [ ] Kindly add this eslint for your code to be guided by best practices.

File: JS/script.js [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+    `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, 
+    but also the leap into electronic typesetting, remaining essent`,
+    technologies: ['html', 'css', 'javaScript'],
+    demo: 'https://chepkok3.github.io/Personal-Portfolio/#',
+    source: 'https://github.com/chepkok3/Personal-Portfolio',
+  },
+];
+
+const popupWindow = document.createElement('div');
+popupWindow.classList.add('popup-wind');
```
Comment (Olamarx) [inline JS/script.js]: - [ ] When the popup is clicked, only one data is popping up throughout. I would advise you to make it pop the very clicked project.
- [ ] Upon clicking it, it pops a blank space until I scroll. I would advise you to make it appear where it is clicked.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @yodit93,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- GitHub flow ✔️ 
- Popup functionalities ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @yodit93 👍 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters check passed ✔️ 
Good commit messages ✔️ 
GitHub flow observed 😄 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (yodit93) [reply .github/workflows/linters.yml:59]: Ok, Thanks.

File: JS/script.js (hunk 2 above)
Comment (yodit93) [reply JS/script.js]: Thank you.
Fixed.

**Pull request merged**